testonly/hammer/hammer.go

```diff
@@ -165,8 +166,26 @@ func (c MapConfig) String() string {
 		c.MapID, c.EPBias, c.Operations, c.EmitInterval, c.RetryErrors)
 }
 
+// SetULimit calls `ulimit -n` to set the maximum number of open files.
```

Comment:
Method doesn't invoke [ulimit(3)](http://man7.org/linux/man-pages/man3/ulimit.3.html), it invokes [setrlimit(2)](http://man7.org/linux/man-pages/man2/setrlimit.2.html)